src/ontology/mondo-edit.obo

```diff
@@ -358597,7 +358435,6 @@ synonym: "hereditary small intestine neuroendocrine neoplasm" EXACT [MONDO:patte
 xref: Orphanet:456333 {source="MONDO:equivalentTo"}
 xref: UMLS:CN237770 {source="MONDO:equivalentTo"}
 xref: UMLS:CN847586 {source="MONDO:equivalentTo"}
-is_a: MONDO:0018538 {source="Orphanet:456333"} ! inherited digestive cancer-predisposing syndrome
```

Comment:
good question. Yes, it's okay because it it has the logical axiom (equivalence axiom), it has inferred parents. If it was orphaned, the QC check would have caught that and the QC would have failed

Reply:
Thanks, I was not clear whether we decided to have a direct `is_a` or if only inferred parents are fine. Would this cause issues for users who only use the asserted version?

Reply:
Users shouldn't use the asserted version. There are other classes that don't have asserted parents.